app/models/miq_worker/runner.rb

```diff
@@ -32,16 +32,12 @@ def self.corresponding_model
     parent
   end
 
-  def self.interrupt_signals
```

Comment:
I believe this was added for easy stubbing of the method for testing but it looks like it's not used anymore so 🔥 🗑 🚽